packages/lesswrong/components/tagging/TagFlagItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import { TagFlags } from "../../lib";
+import { Tags } from "../../lib/collections/tags/collection";
+import { useMulti } from "../../lib/crud/withMulti";
+import { useSingle } from "../../lib/crud/withSingle";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import classNames from 'classnames';
+import { useHover } from "../common/withHover";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import Card from "@material-ui/core/Card";
+import { commentBodyStyles } from "../../themes/stylePiping";
+
+const styles = theme => ({
+  root: {
+    ...theme.typography.commentStyle,
+    padding: 4,
+    margin: 4,
+    borderRadius: 5,
+    backgroundColor: 'rgba(0,0,0,0.1)',
+    display: 'inline-block'
+  }, 
+  black: {
```

Comment:
Would prefer if these three styles (black, white, grey) were named semantically rather than named for their color.

Reply:
Yeah, I did that at first, but then I kept inverting their semantic meaning in different contexts, so I gave up. Specifically, in the edit form, the grey state is the selected state and the white state is the unselected state. But on the Dashboard the grey state is the unselected state and the black state is the selected state.